### PR TITLE
RadioGroup: update demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 1]
```diff
@@ -1 +1,17 @@
-The RadioGroup is&nbsp;a&nbsp;UI component that contains a&nbsp;set of&nbsp;radio buttons and allows an&nbsp;end user to&nbsp;make a&nbsp;single selection from the set. This demo illustrates how to&nbsp;change the component layout, define a&nbsp;custom template and handle the value change event.
\ No newline at end of file
+The RadioGroup component contains a set of radio buttons and allows users to make a single selection from the set. This demo illustrates how to bind component to data, change the component layout, define a custom template, and handle the value change event.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]: 1. Be simpler.
2. Mind the articles.

```suggestion
The RadioGroup component contains a set of radio buttons and allows users to select one item from it. This demo illustrates how to bind the component to data, change the component layout, define a custom template, and handle the value change event.
```

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+## Bind RadioGroup to Data
+
+You can display RadioGroup items from the [items](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/items/) array or a [dataSource](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#dataSource). If your data is an array of objects, use the [displayExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#displayExpr) and the [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) properties. [displayExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#displayExpr) specifies a data source field that supplies texts for buttons. [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) specifies a data source field that supplies values to the value property when users select a button. Leave [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) unspecified if you need to supply the entire data object to the value property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]:
```suggestion
You can display RadioGroup items from the [items](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/items/) array or a [data source](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#dataSource). If your data is an array of objects, use the [displayExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#displayExpr) and [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) properties. [displayExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#displayExpr) specifies a data source field that contains button captions. [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) specifies a data source field that supplies values to the value property when users select a button. Leave [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) unspecified if you need to supply the entire data object to the value property.
```

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+## Configure the Layout
+
+The RadioGroup supports horizontal (default for tablets) and vertical (default for other devices) layouts. To change the layout for all types of devices, specify the [layout](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#layout) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]:
```suggestion
The RadioGroup supports horizontal (default for tablets) and vertical (default for other devices) layouts. To change the layout for all device types, specify the [layout](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#layout) property.
```

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 4]
```diff
@@ -1 +1,17 @@
...
+
+## Customize Item Appearance
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]:
```suggestion
## Customize the Item Appearance
```

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 5]
```diff
@@ -1 +1,17 @@
...
+
+Define the [itemTemplate](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#itemTemplate) to customize item appearance. To customize an individual item, specify the [template](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/items/#template) property of the item.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]: 1. Be simpler.
2. Mind the articles.

```suggestion
Define the [itemTemplate](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#itemTemplate) to customize the item appearance. To customize an individual item, specify its [template](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/items/#template) property.
```

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 6]
```diff
@@ -1 +1,17 @@
...
+
+## Handle the Value Change
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]:
```suggestion
## Handle the Value Change Event
```

File: JSDemos/Demos/RadioGroup/Overview/description.md [hunk 7]
```diff
@@ -1 +1,17 @@
...
+
+The RadioGroup supports horizontal (default for tablets) and vertical (default for other devices) layouts. To change the layout for all types of devices, specify the [layout](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#layout) property.
+
+## Customize Item Appearance
+
+Define the [itemTemplate](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#itemTemplate) to customize item appearance. To customize an individual item, specify the [template](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/items/#template) property of the item.
+
+## Handle the Value Change 
+
+You can set the [value](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#value) property to one of the data source items to specify a predefined selection. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#onValueChanged) function to handle the [value](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#value) change.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/RadioGroup/Overview/description.md]:
```suggestion
You can set the [value](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#value) property to one of the data source items to specify a predefined selection. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#onValueChanged) function to handle [value](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#value) changes.
```

File: JSDemos/Demos/RadioGroup/Overview/description.md (hunk 4 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/RadioGroup/Overview/description.md]: I've heard that the title rules are a bit different and you can omit the articles. I tried to google it, but only found [this article](https://en.wikipedia.org/wiki/Headline#Headlinese). And I've been omitting some articles in the headings and article titles for a pretty long time and haven't even questioned it. The question is: _do we have any guidlines or rules about the titles?_ I found only the title case rule and "how to name the article" in the guidelines, but there's nothing about the grammar.

Comment (dmitry-eliseev-devexpress) [reply JSDemos/Demos/RadioGroup/Overview/description.md]: Alas, no, we do not have one yet. However, I prefer to follow the same rules we use for the rest of our texts. Either way, the thing you did here does not look consistent. First you say in a title:

> Configure **the** Layout

Then, you omit an article in the next title:

> Customize Item Appearance

The Item Appearance literally means "the appearance of all items in general", therefore, it suits the header well.

Comment (vladaskorohodova) [reply JSDemos/Demos/RadioGroup/Overview/description.md]: Okay, got you!

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**